authors/apps/articles/tests/test_like_or_dislike_comments.py

```diff
@@ -0,0 +1,208 @@
+from .base import BaseTest
+from authors.apps.articles.models import Article
+from rest_framework.views import status
+import json
+
+
+class CommentsLikeDislikeTestCase(BaseTest):
+    """test class for liking and disliking comments """
+
+    def get_token(self):
+        response = self.client.post(self.SIGN_UP_URL, self.fav_test_user, format='json')
+        response = self.client.post(self.SIGN_IN_URL, self.fav_test_user, format='json')
+        token = response.data['token']
+        return token
+
+    def create_article(self, token, article):
+        """ Method to create an article"""
+        return self.client.post(self.ARTICLES, self.test_article_data,
+                                HTTP_AUTHORIZATION='Bearer ' + token, format='json')
+
+    def create_comment(self, token, slug, test_comment_data):
+        """ Method to create an article then comment"""
+        self.client.post(self.ARTICLES, self.test_article_data,
+                         HTTP_AUTHORIZATION='Bearer ' + token, format='json')
+        return self.client.post('/api/articles/test-title12/comment/', self.test_comment_data,
+                                HTTP_AUTHORIZATION='Bearer ' + token, format='json')
+
+    def test_like_comment(self):
+        """Test test the liking of a comment"""
+        token = self.get_token()
+        article_data = {
+            "title": "test title",
+            "body": "This is me testing",
+            "description": "testing",
+            "time_to_read": 1,
+            "tags": ["TDD"]
+        }
+
```

Comment:
since you're importing from .base you can use the article data defined there instead of having to define another set of data

Reply:
Check for changes